assets/queries/dockerfile/healthcheck_instruction/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].command
+  not contains(resource, "healthcheck")
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("%s", [resource[0].Cmd]),
+                "issueType":		"MissingAttribute",  #"MissingAttribute" / "RedundantAttribute"
+                "keyExpectedValue": "Dockerfile contains instruction 'HEALTHCHECK'",
+                "keyActualValue": 	"Dockerfile doesn't contain instruction 'HEALTHCHECK'"
+              }
+}
+
+contains(cmd, elem) {
+  cmd[_].Cmd = elem
```

Comment:
Probably would be better to use the contains() function here, and don't forget about the case sensitivity

Reply:
Can't use contains() since I'm checking an array, not a string for an element, case sensitivity will not be a problem here because our parser, parses commands to lowercase